### DISCUSSION TITLE
dolphin集群注册zookeeper时部分组件超时

### DISCUSSION THREAD

**Discussion by huotianyu-CAa** [apache/dolphinscheduler#13919] [Show and tell]
dolphin 3.1.4
zookeeper 3.8.0

问题描述：集群启动后，过一会在查看集群状态，发现部分组件状态为“STOP”（master/worker/alert/api都有可能发生）。日志显示是连接zookeeper集群超时。

问题解决：之前dolphin配置文件中zookeeper集群配置的是ip地址，偶然间在discussions中看到一个帖子说只能通过hostname注册。所以就尝试修改配置文件的zookeeper集群由ip改为hostname。结果就成功了。

**Comment by huotianyu-CAa** [comment]
只能说 amazing

**Comment by shiqin-zhang** [comment]
好吧，原来这样，我也碰到类似的报错，很郁闷不知道怎么解决，你说的方法我准备最近抽空试下。

**Reply by shiqin-zhang** [reply]
尝试了，确实是这样的。而且感觉ds对多网卡支持的文档不够详细，我只保留一张网卡，才集群部署成功。
